site/styles/_buttons.scss

```diff
@@ -200,6 +200,26 @@
   }
 }
 
+.s72-btn.s72-btn-unsubscribe,
+.s72-btn.s72-btn-updatecreditcard {
+  background: #3f3f3f;
+  border: 3px solid #3f3f3f;
+  min-width: 140px;
+  @include hover-focus {
+    background: $gray-700;
+    border-color: $gray-700;
+  }
+}
+
+.s72-btn.s72-btn-explore-subs {
+  background: #1aa3ff;
+  border: 3px solid #1aa3ff;
+  min-width: 140px;
+  @include hover-focus {
+    background: #55bbff;
+    border-color: #55bbff;
+  }
+}
```

Comment:
This makes me go "hmm" - are these colours only actually used on the subs page? Or do we need to use colours defined elsewhere? This might be a question to go back to Design on.

Reply:
+1

Reply:
Sorry should of marked this convo as resolved. This was updated recently, hence the outdated file.